src/app/restaurant/restaurant.component.css

```diff
@@ -1,4 +1,5 @@
-.restaurant, .container, .singleReview {
+.container,
+.singleReview {
```

Comment:
Class names should be kebab case (e.g. `single-review`). Also there's no need for the 'single' numerization in the name, `review` is just fine and simpler.